react-app/src/components/bpm-calculator.tsx

```diff
@@ -108,108 +110,102 @@ export const BPMCalculator: React.FC = () => {
 
   return (
     <div className="calculator">
-      <fieldset className="fieldset">
-        <legend>BPM Controls</legend>
-
-        <div
-          style={{
-            display: "flex",
-            flex: "1 0 auto",
-            flexDirection: "row",
-            justifyContent: "space-between",
-            padding: "5px",
-          }}
-        >
-          <div style={{ alignItems: "center", display: "inline-flex" }}>
+      <div className="calculator-top">
+        <div className="calculator-bpm fieldset">
```

Comment:
```suggestion
        <fieldset className="calculator-bpm fieldset">
```

I want to keep this as semantic HTML5 for accessibility reasons. Furthermore, changing this to a `<div>` sort of defeats the purpose of the `<legend>`.

For more information, see <https://developer.mozilla.org/en-US/docs/Web/HTML/Element/legend>

Reply:
@vapurrmaid Ahhh I knew this would come back to haunt me. Fieldset doesn't support flex, but I'm adding a div with flex within fieldset, keeping legend as the direct descendant of fieldset. Good thing you're baking accessibility in!